certificate-service/integration-certificate-print-service/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id 'java'
+}
+
+group = 'se.inera.intyg.certificateservice'
+version = '0.0.1-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.apache.pdfbox:pdfbox:3.0.2'
```

Comment:
Do we need to use pdf-box in this module? If we do then we shouldn't hardcode version.